### PR TITLE
fix: modification of "model service's resource"

### PR DESCRIPTION
**Pull request by yomybaby**
### TL;DR

This PR resolves the issue where the endpoint's resource allocation values are not displayed when modifying the model service, and the first allocatable preset is automatically selected.

### What changed?
The logic was changed as follows:
- case 1: `allowCustomResourceAllocation = false`
  - Hide resource group select and preset select. and display current value like below: 

  - If the user click the "Change" button, the preset select component is shown with automatic selection of preset
- case 2: `allowCustomResourceAllocation = true`
  - When modifying, the `allocationPreset` field is set to custom.

### CODE REVIEW

**Comment by yomybaby** [comment]
* **#2687** <a href="https://app.graphite.dev/github/pr/lablup/backend.ai-webui/2687?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @yomybaby and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by ironAiken2 — CHANGES_REQUESTED** [review]
It works fine, but it seems like there needs to be some consideration for when the maximum quota for a resource group has changed.

[If the maximum allocatable amount is reduced by a resource policy from what it was before].

**Comment by yomybaby** [comment]
In that case, the user must change the value to be smaller than the slider max. Is the current behavior not like that?

**Review by ironAiken2 — CHANGES_REQUESTED** [review]
Yes, that's how the behavior is implemented now. 

However, if the user doesn't modify the resource, the request can be sent with an existing set value (greater than the resource policy).

and please resolve the conflict!

**Review by ironAiken2 — APPROVED** [review]
LGTM
